modules/boltlib/lib/puppet/datatypes/target.rb

```diff
@@ -1,7 +1,7 @@
 Puppet::DataTypes.create_type('Target') do
```

Comment:
I don't think we have any tests that use this.

Reply:
no I don't think people were really using it before but we should probably create some minimal integration tests at least.